lib/Doctrine/ORM/Mapping/Driver/AnnotationDriver.php

```diff
@@ -277,6 +277,8 @@ public function loadMetadataForClass($className, ClassMetadata $metadata)
         /* @var $property \ReflectionProperty */
         foreach ($class->getProperties() as $property) {
             if ($metadata->isMappedSuperclass && ! $property->isPrivate()
```

Comment:
I believe the time has come to move the whole expression of this if condition into a function :-)